packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-table-manager-lambda/amplify-table-manager-handler.ts

```diff
@@ -25,24 +30,100 @@ const notFinished: AWSCDKAsyncCustomResource.IsCompleteResponse = {
   IsComplete: false,
 };
 
+// #region Entry points
+export const onEvent = cfnResponse.safeHandler(onEventHandler);
+export const isComplete = cfnResponse.safeHandler(isCompleteHandler);
 /**
- * Util function to log especially for nested objects and arrays
- * @param msg
- * @param other arrays of arguments to be logged
+ * Handler for requests sent from CloudFormation for custom resource.
+ * This is the entry point of the custom resource.
+ *
+ * Note: This is adapted from the AWS CDK's provider framework.
+ * https://github.com/aws/aws-cdk/blob/11621e7/packages/aws-cdk-lib/custom-resources/lib/provider-framework/runtime/framework.ts
+ * @param cfnRequest Request received from CloudFormation
  */
-const log = (msg: string, ...other: any[]) => {
-  console.log(
-    msg,
-    other.map((o) => (typeof o === 'object' ? JSON.stringify(o, undefined, 2) : o)),
-  );
-};
+// eslint-disable-next-line func-style
+async function onEventHandler(cfnRequest: AWSLambda.CloudFormationCustomResourceEvent): Promise<void> {
```

Comment:
nit: why aren't we using arrow function here?

Reply:
Code organization within the file. A function definition allows the entry points to be defined above the handlers they wrap.

The actual entry points invoked by CloudFormation (onEvent) and our state machine (isComplete) are defined [above](https://github.com/aws-amplify/amplify-category-api/pull/2490/files/92d96143e9b879da06817a9967fcef06db28cdc1#diff-f8f8c42c334f8453ee32eb354a7a49ad97ff4f900670f88ce74fecf93e253c48R34-R35).

```ts
export const onEvent = cfnResponse.safeHandler(onEventHandler);
export const isComplete = cfnResponse.safeHandler(isCompleteHandler);
```

I'm happy to move to an arrow function if you feel the cost of differing function styles outweighs the benefit of defining the entry points above them.

Reply:
no strong opinions here. The inconsistent style makes me a little twitchy but I enthusiastically admit that's a me thing.